docs/content/guide/filter.ngdoc

```diff
@@ -109,7 +109,7 @@ as the first argument. Any filter arguments are passed in as additional argument
 function.
 
 The filter function should be a [pure function](http://en.wikipedia.org/wiki/Pure_function), which
-means that it should be stateless and idempotent, and not rely for example on other Angular services.
+means that it should be deterministic and not rely, for example, on other Angular services.
```

Comment:
Also note that in the context, this is giving explanation of "pure function" in the sentence. Pure functions are most certainly not idempotent by necessity (`f(x) = x + 1` is pure but not idempotent), and they can even be stateful in careful cases.